base/docker/scripts/start.sh

```diff
@@ -94,52 +95,56 @@ setup_database() {
         ADB_PORT=55555
     fi
 
-    # Initialize ADB_HOST and ADB_VERIFY_HOSTNAME
-    local ADB_HOST
-    local ADB_VERIFY_HOSTNAME
-    if [ -n "${DB_HOST_PRIVATE:-}" ]; then
-        ADB_HOST=$(/app/wf_argparse.py --type hostname --envar DB_HOST_PRIVATE)
-        ADB_VERIFY_HOSTNAME=false
-    elif [ -n "${DB_HOST_PUBLIC:-}" ]; then
-        ADB_HOST=$(/app/wf_argparse.py --type hostname --envar DB_HOST_PUBLIC)
-        ADB_VERIFY_HOSTNAME="${VERIFY_HOSTNAME_DEFAULT}"
-    elif [ -z "${DB_HOST:-}" ]; then
-        ADB_HOST="localhost"
-        ADB_VERIFY_HOSTNAME=false
-    else
-        local DB_HOST_VAL=$(/app/wf_argparse.py --type hostname --envar DB_HOST )
-        if [ "${DB_HOST_VAL}" == "localhost" ] || [ "${DB_HOST_VAL}" == "127.0.0.1" ] || [ "${DB_HOST_VAL}" == "::1" ]; then
-            ADB_HOST="${DB_HOST_VAL}"
+    if [ -z "${APERTUREDB_KEY:-}" ]; then
+        # Initialize ADB_HOST and ADB_VERIFY_HOSTNAME
+        local ADB_HOST
+        local ADB_VERIFY_HOSTNAME
+        if [ -n "${DB_HOST_PRIVATE:-}" ]; then
+            ADB_HOST=$(/app/wf_argparse.py --type hostname --envar DB_HOST_PRIVATE)
             ADB_VERIFY_HOSTNAME=false
-        else
-            ADB_HOST="${DB_HOST_VAL}"
+        elif [ -n "${DB_HOST_PUBLIC:-}" ]; then
```

Comment:
This looks messy, but should be the current code at https://github.com/aperture-data/workflows/blob/edd61eefc19ec2dc5d44b91dcc885ef5229e26a5/base/docker/scripts/start.sh but indented.